docker/docker-compose.override.yml

```diff
@@ -43,6 +45,5 @@ services:
       && sleep 15
       && cd api/
       && ../node_modules/.bin/sequelize db:migrate
-      && ../node_modules/.bin/sequelize db:seed:all
```

Comment:
@waliens Is that correct?

Reply:
Depends, if you remove it, people launching the docker from scratch won't have data in the database. This command is a problem if you already have data in the database before launching the container. I usually use it when I create a new env for working on a new feature

Reply:
So should we commit it then? :)

Reply:
I would say no :)